fpdf/fpdf.py

```diff
@@ -353,6 +353,7 @@ def __init__(
         self.pdf_version = "1.3"  # Set default PDF version No.
         self.creation_date = datetime.now(timezone.utc)
         self._security_handler = None
+        self._fallback_fonts = None
```

Comment:
```suggestion
        self._fallback_font_ids = None
```
Or `_fallback_font_names`?